ionic/components/app/normalize.scss

```diff
@@ -151,6 +151,11 @@ textarea {
   touch-action: manipulation;
 }
 
+button ion-label,
+a ion-label {
+    pointer-events: none;
+}
+
```

Comment:
@adamdbradley not sure if this should be here (normalise.scss) or in label.scss or in buttons.scss...